queries.sql

```diff
@@ -22,3 +22,93 @@ WHERE name <> 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- First transaction
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
```

Comment:
- [ ] 👋 Hello! Fantastic job on the implementation of the query transaction! 🚀 Please make sure to double-check the modifications, as the previous reviewer suggested 🕵️, before considering any rollback 🔄. This extra step helps ensure that any changes made are thoroughly validated and in line with our requirements 📝. Thank you for your attention to detail! 🙌😊
```suggestion
SELECT species from animals; -- verify that change was made
```

Reply:
Done, thanks for you review 👍😃